fs2/src/main/scala/io/finch/fs2/package.scala

```diff
@@ -0,0 +1,62 @@
+package io.finch
+
+import _root_.fs2.Stream
+import cats.effect.Effect
+import com.twitter.finagle.http.Response
+import com.twitter.io.Buf
+import io.finch.internal.futureToEffect
+import io.finch.streaming.StreamFromReader
+import shapeless.Witness
+
+package object fs2 extends StreamInstances {
+
+  implicit def streamFromReader[F[_] : Effect]: StreamFromReader[Stream, F] = StreamFromReader.instance { reader =>
+    Stream
+      .repeatEval(futureToEffect(reader.read()))
+      .unNoneTerminate
+      .onFinalize(Effect[F].delay(reader.discard()))
+  }
+
+  implicit def streamToJsonResponse[F[_] : Effect, A](implicit
+    e: Encode.Aux[A, Application.Json],
+    w: Witness.Aux[Application.Json]
+  ): ToResponse.Aux[Stream[F, A], Application.Json] = {
+    mkToResponse[F, A, Application.Json](delimiter = Some(ToResponse.NewLine))
+  }
+
+}
+
+trait StreamInstances {
+
+  implicit def streamToResponse[F[_] : Effect, A, CT <: String](implicit
```

Comment:
I believe these two instances can now go under `ToResponse` companion object?

Reply:
Sadly, nope. `ToResponse` exists in the core, there is no `fs2` stuff there

Reply:
This should be possible after #1056.